src/Model/Behavior/UploadBehavior.php

```diff
@@ -94,7 +96,11 @@ public function beforeSave(EventInterface $event, EntityInterface $entity, Array
                 continue;
             }
 
-            if (Hash::get((array)$entity->get($field), 'error') !== UPLOAD_ERR_OK) {
+            if (empty($entity->get($field))) {
```

Comment:
Why doesn't this check for errors any more?

Reply:
It's checked bellow, on line 103. This empty check avoid errors on line 103 when no file is send.